### PR TITLE
Create CSL for Land Degradation & Development

### PR DESCRIPTION
**Pull request by gilrockmendes**

### CODE REVIEW

**Comment by rmzelle** [comment]
Thanks. Could you go through https://github.com/citation-style-language/styles/wiki/Style-Requirements and update your style as necessary?

(the file name should be "land-degradation-and-development.csl"; see https://help.github.com/articles/renaming-a-file/)

File: land-degradation-and-development.csl [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+        <text macro="title"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="author-short">
+    <names variable="author">
+      <name form="short" and="symbol" et-al-min="3" et-al-use-first="1" initialize-with=". "/>
```
Comment (adam3smith) [inline land-degradation-and-development.csl]: remove `et-al-min="3" et-al-use-first="1" ` that's already covered in by the same info under citation.

File: land-degradation-and-development.csl [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+      <et-al font-style="italic"/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+        <text variable="container-title" font-style="italic"/>
```
Comment (adam3smith) [inline land-degradation-and-development.csl]: this seems odd -- if you don't have the same line in the author macro, matching citation to bibliography would be quite tricky.

**Comment by adam3smith** [comment]
See two nits in comments inline, everything else looks good.

**Comment by rmzelle** [comment]
Add in #2261 with some minor changes.

Thanks, @gilrockmendes, and sorry for the long wait.

**Pull request closed without merging**